internal/namespace/manager.go

```diff
@@ -124,3 +139,138 @@ func (nm *namespaceManager) initNamespaces(ctx context.Context, di database.Plug
 	}
 	return nil
 }
+
+func (nm *namespaceManager) validateNamespaceConfig(ctx context.Context, name string, index int, conf config.Section) error {
+	if err := core.ValidateFFNameField(ctx, name, fmt.Sprintf("namespaces.predefined[%d].name", index)); err != nil {
+		return err
+	}
+
+	if strings.ToLower(name) == core.SystemNamespace || strings.ToLower(conf.GetString(coreconfig.NamespaceRemoteName)) == core.SystemNamespace {
```

Comment:
Not sure `ToLower` is needed - our namespaces are case-sensitive, right? Although we could choose to disallow any variation of ff_system here.

Reply:
wasn't sure, I guess since FF_SYSTEM is technically allowed I'll remove `toLower()`